src/htrun/README.md

```diff
@@ -261,31 +261,9 @@ Key-value protocol was developed and is used to provide communication layer betw
 
 # Greentea client API
 
-DUT test API was first introduced in ```mbedmicro/mbed``` project [here](https://github.com/mbedmicro/mbed/tree/master/libraries/tests/mbed/env). After refactoring this functionality was copied and improved in [greentea-client](https://github.com/ARMmbed/mbed-os/tree/master/features/frameworks/greentea-client) module.
+[greentea-client](https://github.com/ARMmbed/greentea-client) is a C++ client library that provides greentea_send_kv API for sending and greentea_parse_kv API to receive key-value pairs with the Greentea host test tool.
 
-* Slave side key-value protocol API, see [here](https://github.com/ARMmbed/mbed-os/blob/master/features/frameworks/greentea-client/greentea-client/test_env.h) for details.
-
-```c++
-// Send key-value pairs from slave to master
-void greentea_send_kv(const char *, const char *);
-void greentea_send_kv(const char *, const int);
-void greentea_send_kv(const char *, const int, const int);
-void greentea_send_kv(const char *, const char *, const int);
-void greentea_send_kv(const char *, const char *, const int, const int);
-
-// Blocking, receive key-value message from master
-int greentea_parse_kv(char *, char *, const int, const int);
-```
-Functions are used to send key-string or key-integer value messages to master. This functions should replace typical ```printf()``` calls with payload/control data to host.
-
-* **Blocking** wait for key-value pair message in input stream:
-
-```c++
-int greentea_parse_kv(char *out_key, char *out_value, const int out_key_len, const int out_value_len);
-```
-
-This function should replace ```scanf()``` used to check for incoming messages from master.
-Function parses input and if key-value message is found load to ```out_key```, ```out_value``` key-value pair. Use ```out_key_size``` and ```out_value_size``` to define out buffers max size (including trailing zero).
+* Slave side key-value protocol API, see [here](https://github.com/ARMmbed/greentea-client/blob/master/include/greentea-client/test_env.h) for details.
 
 # Key-value transport protocol sequence
 
```

Comment:
Shall we remove lines 307 to 331 (line numbers in your new version)? [The code has changed](https://github.com/ARMmbed/greentea-client/blob/08fdb8b68f99c523ee298dea6c93eedb60aef9a8/source/greentea_test_env.cpp#L66-L87) so the snippet isn't accurate anymore. Rather than updating the snippet or adding a link, I think we can omit it altogether because diagram there is enough to explain the htrun log.